### ISSUE TITLE
Rendering now hangs after first notebook runs

### ISSUE THREAD

**Issue by gschivley** [quarto-dev/quarto-actions#99]
I've been using Quarto to create dashboards for a project. The GH action runs on every push, rendering [a series of .qmd files](https://github.com/gschivley/MIP_results_comparison/tree/main/notebooks) to notebooks and then publishing the dashboards to a blog. This has worked amazingly well for several months, but the [action started hanging today](https://github.com/gschivley/MIP_results_comparison/actions). An action that normally takes 12 min has been running for 3 hours with no progress after [ 2/13].

```
Run quarto-dev/quarto-actions/publish@v2
Run git config --global user.email "quarto-github-actions-publish@example.com"
From https://github.com/gschivley/MIP_results_comparison
 * branch            gh-pages   -> FETCH_HEAD
 * [new branch]      gh-pages   -> origin/gh-pages
Rendering for publish:


[ 1/13] blog.qmd

[ 2/13] notebooks/26z-short-base-1000.qmd
Starting python3 kernel...Done
Executing '26z-short-base-1000.ipynb'
  Cell 1/23: ''...Done
  Cell 2/23: ''...Done
  Cell 3/23: ''...Done
  Cell 4/23: ''...Done
  Cell 5/23: ''...Done
  Cell 6/23: ''...Done
  Cell 7/23: ''...Done
  Cell 8/23: ''...Done
  Cell 9/23: ''...Done
  Cell 10/23: ''...Done
  Cell 11/23: ''...Done
  Cell 12/23: ''...Done
  Cell 13/23: ''...Done
  Cell 14/23: ''...Done
  Cell 15/23: ''...Done
  Cell 16/23: ''...Done
  Cell 17/23: ''...Done
  Cell 18/23: ''...Done
  Cell 19/23: ''...Done
  Cell 20/23: ''...Done
  Cell 21/23: ''...Done
  Cell 22/23: ''...Done
  Cell 23/23: ''...Done
```

I can't see anything that would have caused this to happen. Things that I've checked out:

- The [last working action job](https://github.com/gschivley/MIP_results_comparison/actions/runs/7834364939/job/21377399982) and the ones today both run quarto 1.5.9. 
- I was able to run `quarto publish` on my local machine using Quarto 1.5.6 and publish with updated data to GitHub pages. Using that version in the GH action rather than "pre-release" didn't work.
- There were not any changes to the code or data of the [first](https://github.com/gschivley/MIP_results_comparison/blob/main/notebooks/26z-short-base-1000.qmd) or [second](https://github.com/gschivley/MIP_results_comparison/blob/main/notebooks/26z-short-base-200-foresight.qmd) .qmd files that are rendered.

Any help would be appreciated!

**Comment by cscheid** [comment]
We just fixed an issue that might have caused this in 1.5.9. Can you confirm that it goes away in 1.5.10?

**Comment by dragonstyle** [comment]
We just found an issue that was related to itables which only appears in the latest version (released just a couple of days ago). The fix didn't quite make 1.5.10, but it should be in 1.5.11 which should be available within a few minutes.

I've also observed some differences in formatting between Quarto and itable 1.6.4 and 1.7.0 but I haven't yet gotten to the bottom of that. If you see things any notes would be helpful as I try to track it down. Thanks and sorry for the issue!
